solidity/dashboard/src/components/DelegationOverview.jsx

```diff
@@ -50,6 +50,17 @@ const DelegationOverview = ({
   return (
     <section>
       <DelegationHeader type={context} selectedGrant={selectedGrant} />
+      <LoadingOverlay
+        isFetching={areTopUpsFetching}
+        skeletonComponent={<DataTableSkeleton columns={3} />}
+      >
+        <Tile>
+          <TopUpsDataTable
```

Comment:
There is also the issue with disabled **commit top-up** button for newly created top-ups.

https://github.com/keep-network/keep-core/blob/0c5748cd54e8f479a8ffbeedb2a17fe06264591b/solidity/dashboard/src/components/TopUpsDataTable.jsx#L52-L58

I'm not sure what `initializationPeriod` with the value of 1 should do, but i assume that after 1 second it should enable the button. We should probably somehow inform the component that this delegation is ready to be committed, because right now we have to refresh the page to commit a top-up.

A quick idea that pops into my mind is that we can maybe add `readyToBeCommitted` property to the topUp objects that are in the redux state? 🤔 And then just disable/enable the button based on that property?